c2rust-analyze/src/type_desc.rs

```diff
@@ -44,11 +44,9 @@ pub fn perms_to_desc(perms: PermissionSet, flags: FlagSet) -> (Ownership, Quanti
         Ownership::Imm
     };
 
-    let qty = if perms.contains(PermissionSet::OFFSET_SUB) {
+    let qty = if perms.contains(PermissionSet::OFFSET_SUB) || perms.contains(PermissionSet::OFFSET_ADD) {
```

Comment:
This change wrongly implies that `OFFSET_SUB` and `OFFSET_ADD` should be handled the same way, and leaving the comment intact is highly misleading.

Reply:
Ok I'll change it back and suppress it with a comment saying they will be different in the future.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/563/commits/afbee3e5ac4cd51e00de3501e91e08992d368b8d.